fusor-ember-cli/app/templates/openshift/openshift-nodes.hbs

```diff
@@ -37,44 +37,90 @@
 
     </p>
 
-    {{button-selection label=1
-                       value=1
-                       groupValue=model.numNodes
-                       changed="numNodesChanged"
-                       disabled=isOver1Capacity}}
-    {{button-selection label=2
-                       value=2
-                       groupValue=model.numNodes
-                       changed="numNodesChanged"
-                       disabled=isOver2Capacity}}
-    {{button-selection label=3
-                       value=3
-                       groupValue=model.numNodes
-                       changed="numNodesChanged"
-                       disabled=isOver3Capacity}}
-    {{button-selection label=4
-                       value=4
-                       groupValue=model.numNodes
-                       changed="numNodesChanged"
-                       disabled=isOver4Capacity}}
-    {{button-selection label=5
-                       value=5
-                       groupValue=model.numNodes
-                       changed="numNodesChanged"
-                       disabled=isOver5Capacity}}
-    <!--<div style='color: #999; font-style: italic;'>
-      Not Highly Available
-    </div>-->
+<div class='row'>
+  <div class='col-xs-8 col-sm-8 col-md-7 col-lg-5 ose-number-node-buttons'>
+    <div class='ose-number-buttons-block'>
+      <span class='ose-node-label'>Master Nodes</span>
+      {{button-selection label=1
+                         value=1
+                         groupValue=numMasterNodes
+                         changed="numMasterNodesChanged"}}
+      {{button-selection label=2
+                         value=2
+                         groupValue=numMasterNodes
+                         changed="numMasterNodesChanged"
+                         disabled=is2MasterOverCapacity}}
+      {{button-selection label=3
+                         value=3
+                         groupValue=numMasterNodes
+                         changed="numMasterNodesChanged"
+                         disabled=is3MasterOverCapacity}}
+      {{button-selection label=4
+                         value=4
+                         groupValue=numMasterNodes
+                         changed="numMasterNodesChanged"
+                         disabled=is4MasterOverCapacity}}
+      {{button-selection label=5
+                         value=5
+                         groupValue=numMasterNodes
+                         changed="numMasterNodesChanged"
+                         disabled=is5MasterOverCapacity}}
+      {{#if isCustomNumMasterNodes}}
+        {{input value=numMasterNodes
+                class='ose-custom-input'
+                placeholder='Enter'}}
+      {{else}}
+        <a {{action 'showCustomNumMasterNodes'}}>advanced</a>
+      {{/if}}
+
+    </div>
+    <div class='ose-number-buttons-block'>
+      <span class='ose-node-label worker-nodes-label'>Worker Nodes</span>
+      {{button-selection label=1
```

Comment:
Maybe we can iterate a structure on the controller here to address the repetition?